### DISCUSSION TITLE
Person Card "Away" vs "not_home"

### DISCUSSION THREAD

**Discussion by JSaterdalen** [UI-Lovelace-Minimalist/UI#659] [Q&A]
Is it possible to have the Person Card display "Home" and "Away", when not displaying a different zone?

The documentation shows "Present" and "Absent", so it seems like it must be customizable, somehow. My cards display "home" and "not_home", which don't look as polished. 

![image](https://user-images.githubusercontent.com/6761188/168125824-bf1fd041-9117-4b5d-9afd-4d854cbee003.png)

**Comment by 13robin37** [comment]
The Person Card (card_person) should display that already. What does your config for those cards look like?
